testColorCodePair.cpp

```diff
@@ -0,0 +1,38 @@
+#include "testColorCodePair.h"
+
+//function to test pair number with color code pair.
+void UnitTest::testNumberToPair(int pairNumber,
+    telecomColorCoder::MajorColor expectedMajorColor,
+    telecomColorCoder::MinorColor expectedMinorColor)
+{
+    telecomColorCoder::ColorPair colorPair =
+        telecomColorCoder::fetchColorFromPairNumber(pairNumber);
+    std::cout << "Got pair " << colorPair.toColorPairString() << std::endl;
+    assert(colorPair.fetchMajorColor() == expectedMajorColor);
+    assert(colorPair.fetchMinorColor() == expectedMinorColor);
+}
+
+//function to test pair number with reference manual color-code pair.
+void UnitTest::testNumberToColorPairRefManual(int pairNumber,
+    telecomColorCoder::MajorColor expectedMajorColor,
+    telecomColorCoder::MinorColor expectedMinorColor)
+{
+    telecomColorCoder::ColorPair colorPair =
+        telecomColorCoder::fetchColorFromPairNumber(pairNumber);
+    std::cout << "Pair " << pairNumber << " also called " 
```

Comment:
the test method indicates that it is testing the ref manual string.
But the verification of string is not done here. std::cout can only be verified manually and not programmatically.
Please modify the test to verify the output string.

Reply:
Fixed